AGENT-PLAN/00-QUICK-START.md

```
@@ -0,0 +1,102 @@
+# Epicourier Agent Quick Start
+
+## 🚀 For New Agent Sessions
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)